extensions/adobe/experience/decisioning/eventmetrics.schema.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I don't see the point in this and it not int he same style as other uses of Measures. Just put them directly in the target locations. The semantic of being a Metric is defined by the type:measure.

Reply:
I will delete this schema and introduce new ones for metric definitions.